Samples/EventHorizon/Server/Bootloader.cs

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) Dolittle. All rights reserved.
```

Comment:
This looks like a copy / paste. Instead you could just refer to the Runtime.Server project and call this.